app/src/main/cpp/src/components/transport_manager/src/bluetooth_le/ble_server.cc

```diff
@@ -0,0 +1,92 @@
+#include "transport_manager/bluetooth_le/ble_server.h"
+#include <thread>
+#include "utils/logger.h"
+
+SDL_CREATE_LOG_VARIABLE("Ble_Server")
+
+namespace{
+    const char* socket_name =  "./localBleWriter";
+
+    char buffer[256];
```

Comment:
@ychernysheva I would use for a separate constant for a buffer size here and at line 15

Reply:
And I believe that buffer size needs to be equal MTU?

Reply:
But i'm not sure how to transfer this data from Java side, if needed?

Reply:
May I suggest to store MTU value in some external file, for example in ini file, to read this value from Java and Native side?

Reply:
@ychernysheva let's keep it simple for this PR. Just declare a const variable at the beginning of this file and use it. We will think about aligning with MTU later in a separate PR if needed.

Reply:
Done in d331af8